internal/controller/cloud-control/vpcpeering_azure_test.go

```diff
@@ -67,6 +67,14 @@ var _ = Describe("Feature: KCP VpcPeering", func() {
 				Should(Succeed())
 		})
 
+		By("Then KCP VpcPeering state is Initiated", func() {
+			Eventually(LoadAndCheck).
+				WithArguments(infra.Ctx(), infra.KCP().Client(), obj,
+					NewObjActions(),
+					HavingState("Initiated"),
```

Comment:
Why not `cloudcontrolv1beta1.VirtualNetworkPeeringStateInitiated` as in the reconciler?